source/adapters/level_zero/platform.cpp

```diff
@@ -48,7 +48,8 @@ UR_APIEXPORT ur_result_t UR_APICALL urPlatformGet(
   }
 
   if (getenv("SYCL_ENABLE_PCI") != nullptr) {
-    urPrint("WARNING: SYCL_ENABLE_PCI is deprecated and no longer needed.\n");
+    logger::warning(
+        "WARNING: SYCL_ENABLE_PCI is deprecated and no longer needed.");
```

Comment:
Remove the `WARNING` word as the logger should provide this information.

Reply:
Done